packages/client/clientSchema.graphql

```diff
@@ -23,6 +23,7 @@ extend type PokerMeeting {
   localStage: NewMeetingStage!
   showSidebar: Boolean!
   githubSearchQuery: GitHubSearchQuery!
+  gitlabSearchQuery: String
```

Comment:
For now, this is just a query string, but it'll be updated to an object like `GitHubSearchQuery` in a later PR when we add the project filter ids